### PR TITLE
 Update ioredis type definitions (add pfcount, pfadd, pfmerge interfaces)

### PR DESCRIPTION
**Pull request by aliarham11**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://redis.io/commands/pfadd, https://redis.io/commands/pfcount, https://redis.io/commands/pfmerge
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by plantain-00** [comment]
👍

**Review by chris-codaio — APPROVED** [review]
:+1:

File: types/ioredis/index.d.ts [hunk 1]
```diff
@@ -335,6 +336,12 @@ declare module IORedis {
         hscan(args: any[], callback?: ResCallbackT<any>): any;
         zscan(...args: any[]): any;
         zscan(args: any[], callback?: ResCallbackT<any>): any;
+        pfmerge(...args: any[]): any;
+        pfmerge(args: any[], callback?: ResCallbackT<any>): any;
```
Comment (andy-ms) [inline types/ioredis/index.d.ts]: The second overload isn't going to be reachable since `...args: any[]` accepts anything.

Comment (aliarham11) [reply types/ioredis/index.d.ts]: It's done, thanks for the review 👍

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**